packages/prisma-client/src/types.ts

```diff
@@ -2,9 +2,11 @@ import type { Asset as DbAsset, Project as BaseProject } from "@prisma/client";
 export { Location } from "@prisma/client";
 export type { InstanceProps, User, Breakpoints } from "@prisma/client";
 
-export type Asset = DbAsset & {
+export type Asset = Omit<DbAsset, "width" | "height"> & {
```

Comment:
We discussed to move all augmentation to entity-specific folders and live prisma client as-is

Reply:
Moved this to another place in the uploader pckage